mdsuite/project/project.py

```diff
@@ -124,22 +126,21 @@ def __str__(self):
 
     def add_experiment(
         self,
-        experiment: str = NoneType,
+        experiment: str = NoneType,  # todo: rename to sth like 'name'
```

Comment:
Yes! We should adress this either in an issue or here directly. I like `name` because it is clear that it is the name of the experiment